integration/testproject/src/wildcard/index.ts

```diff
@@ -0,0 +1,16 @@
+import * as foo from './foo';
+import * as b from './b';
+
+const x = foo.x;
+const {y} = foo;
+const {z: {a}} = foo;
+const w = foo['w'];
+
+console.log(x, y, a, w);
+console.log(b[Math.random() < 0.5 ? 'a' : 'b']);
```

Comment:
`trackWildcardUses` isn't smart enough to track this pattern, so it falls back to marking all of `b` as definitely used.